beetsplug/deezer.py

```diff
@@ -122,15 +122,23 @@ def _get_track(self, track_data):
         artist, artist_id = self.get_artist(
             track_data.get('contributors', [track_data['artist']])
         )
+
+        track_position = None
+        disk_number = None
+        if 'track_position' in track_data:
+            track_position = track_data['track_position']
+        if 'disk_number' in track_data:
+            disk_number = track_data['disk_number']
```

Comment:
These conditionals can be simplified to calls to `.get()`, like `track_position = track_data.get(['track_position')`.

Reply:
Oh right of course 🤦‍♂️